anesthetic/samples.py

```diff
@@ -764,6 +764,10 @@ def recompute(self, logL_birth=None, inplace=False):
 
         samples.tex['nlive'] = r'$n_{\rm live}$'
         samples.beta = samples._beta
+
+        if np.any(pandas.isnull(samples.logL)):
+            warnings.warn("Null value encountered in logL.", RuntimeWarning)
```

Comment:
I would prefer `np.isnan` over `pandas.isnull` and `NaN` over `Null`, which I think are more pythonic. The pandas "null" comes from R...

Reply:
Actually, pandas can apparently do more... So I'll revise my comment to: I would prefer `pandas.isna` over `pandas.isnull` and `NaN` over `Null`.